github/github.go

```diff
@@ -841,9 +841,9 @@ An ErrorResponse reports one or more errors caused by an API request.
 GitHub API docs: https://docs.github.com/en/rest/#client-errors
 */
 type ErrorResponse struct {
-	Response *http.Response // HTTP response that caused this error
-	Message  string         `json:"message"` // error message
-	Errors   []Error        `json:"errors"`  // more detail on individual errors
+	Response *http.Response `json:"omitempty"` // HTTP response that caused this error
```

Comment:
I've never seen a bare "omitempty" before, so can we please name it as "response"?

```suggestion
	Response *http.Response `json:"response,omitempty"` // HTTP response that caused this error
```

Reply:
I'm fine adding the name of the field. But I wonder in this case if it might needs to be `Response` to maintain compatibility with the current code.

Reply:
You are saying it should be "Response" with a capital-R ?
When you say "compatibility with the current code", do you mean the "go-github" repo?
If so, I'm not sure why we would need to capitalize the R, and I would think the lower-case version should be fine.

Alternatively, another approach might be to simply use `json:"-"` and always leave it out... but I'll let you decide if that is the best option for compatibility with your mocking package.

Reply:
Hi Glenn, I actually liked the idea of going with `json:"-"`. Let me test a few things and I will come back to you.

Thanks for the swift responses!

Reply:
Hi @gmlewis , it seems the `json:"-"`  should work just fine. I will update the PR soon.

Reply:
done :)